src/main/scala/mesosphere/marathon/state/Migration.scala

```diff
@@ -116,11 +119,11 @@ class Migration @Inject() (
   }
 
   private def changeTasks(fn: InternalApp => InternalApp): Future[Any] = {
```

Comment:
Please add a Migration, which renames:
`tasks:appid:appid.uuid` to `tasks:appid.uuid`